_shared_assets/themes/nextcloud_com/static/styles.css

```diff
@@ -1108,7 +1108,6 @@ a.headerlink {
 }
 
 #menu-support > ul > .current {
-    border: 1px solid #999 !important;
```

Comment:
Removing this will remove the left border in the menu for the active entry:

Without:

![bildschirmfoto 2017-10-23 um 22 46 57](https://user-images.githubusercontent.com/245432/31912424-1e1857b4-b844-11e7-9085-f51009719c03.png)

With:

![bildschirmfoto 2017-10-23 um 22 46 53](https://user-images.githubusercontent.com/245432/31912430-227c75d8-b844-11e7-8c8b-0836145aa9ad.png)